File: java/beginner/001_HelloWorld.java
public class HelloWorld {

    public static void main(String[] args) {
        System.out.println("Hello Java World");

        String msg = greet("Alex");
        System.out.println(msg);

        sayHello("Sam");
        sayHello("Jamie");

        int result = add(3, 5);
        System.out.println("3 + 5 = " + result);
    }

    public static String greet(String name) {
        return "Hello " + name;
    }

    public static void sayHello(String name) {
        System.out.println("Hi " + name);
    }

    public static int add(int a, int b) {
        return a + b;
    }
}

File: csharp/beginner/001_HelloWorld.cs
// [1] Hello World와 메서드 - Hello World & Methods
// 레벨: 1 | C#의 기본 구조와 메서드를 배웁니다

using System;

class Program
{
    static void Main(string[] args)
    {
        Console.WriteLine("Hello CSharp");

        string msg = Greet("Alex");
        Console.WriteLine(msg);

        Say("Sam");
        Say("Lee");

        int result = Add(3, 5);
        Console.WriteLine("3 + 5 = " + result);
    }

    static string Greet(string name)
    {
        return "Hello " + name;
    }

    static void Say(string name)
    {
        Console.WriteLine("Hi " + name);
    }

    static int Add(int a, int b)
    {
        return a + b;
    }
}
